src/test/java/com/google/javascript/clutz/partial/missing_logger_dot_level.js

```diff
@@ -0,0 +1,10 @@
+goog.provide('goog.log.Level');
+
+goog.require('goog.debug.Logger');
+
+
```

Comment:
Can you add a comment here about the sort of output you expect?  (That is, that Level is an alias for Logger.Level in the d.ts.)